src/SUMMARY.md

```diff
@@ -76,7 +76,8 @@
 - [Welcome](welcome-day-2.md)
 - [Pattern Matching](pattern-matching.md)
   - [Matching Values](pattern-matching/match.md)
-  - [Destructuring](pattern-matching/destructuring.md)
+  - [Destructuring Structs](pattern-matching/destructuring-structs.md)
+  - [Destructuring Enums](pattern-matching/destructuring-enums.md)
```

Comment:
#1464 suggests making these sub-slides, but maybe the two slides makes more sense here?

Reply:
From last 2 PRs, I have been making it into different slides. At start we used to make them as subslides ,But Not sure about the trasaction.

Reply:
Sounds good -- thanks for updating the timing!

Reply:
Thanks!

Reply:
Need a update on #1942 PR comment.